target/linux/ramips/dts/mt7621_d-team_pbr-m1.dts

```diff
@@ -87,6 +74,33 @@
 		compatible = "gpio-beeper";
 		gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
 	};
+
+	reg_power_usb2: regulator1 {
+		compatible = "regulator-fixed";
+		regulator-name = "power_usb2";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		gpios = <&gpio 22 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+	};
+
+	reg_power_usb3: regulator2 {
+		compatible = "regulator-fixed";
+		regulator-name = "power_usb3";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		gpios = <&gpio 25 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+	};
+
+};
+
+&xhci_ehci_port1 {
+	vbus-supply = <&reg_power_usb3>;
+};
+
+&ehci_port2 {
+	vbus-supply = <&reg_power_usb2>;
```

Comment:
Please move these vbus-supply properties into xhci node. ref:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/usb/mediatek%2Cmtk-xhci.yaml

The two regulators can be merged into one by `gpios = <&gpio 22 GPIO_ACTIVE_HIGH>, <&gpio 25 GPIO_ACTIVE_HIGH>;`